.golangci.yml

```diff
@@ -0,0 +1,35 @@
+version: "2"
+linters:
+  disable:
+    - errcheck      # TODO(rsned): Enable once outstanding lint bugs are fixed.
+    - mnd
+    - paralleltest
+    - staticcheck   # TODO(rsned): Enable once outstanding lint bugs are fixed.
+    - testpackage
+    - unused
+  exclusions:
+    generated: lax
+    presets:
```

Comment:
From what I can tell, we don't actually violate any of these false-positive presets today.   Would it be better to exlcude them until they cause a problem?

Reply:
I'm still seeing the presets.

Reply:
removed the presets